mapbox/__init__.py

```diff
@@ -9,3 +9,4 @@
 from .services.surface import Surface
 from .services.static import Static
 from .services.uploads import Uploader
+from .services.analytics import Analytics
```

Comment:
imports should be sorted alphabetically, so this one goes to the top of the list.